pkg/pub_integration/test/fake_pub_server_test.dart

```diff
@@ -37,12 +39,38 @@ void main() {
     tearDownAll(() async {
       await tempDir?.delete(recursive: true);
       await fakePubServerProcess?.kill();
+      httpClient.close();
     });
 
     test('standard integration', () async {
+      final inviteUrlFuture = fakePubServerProcess
+          .waitForLine((line) => line.contains('/admin/confirm/new-uploader/'));
+      Future inviteCompleterFn() async {
+        final pageUrl = await inviteUrlFuture.timeout(Duration(seconds: 30));
+        final pageUri = Uri.parse(pageUrl);
+        final confirmUrl = pageUri.replace(
+          scheme: 'http',
+          host: 'localhost',
+          port: fakePubServerProcess.port,
+        );
+        print('Requesting confirm URL $confirmUrl');
+        final pageRs = await httpClient.get(confirmUrl);
+        final acceptLine = pageRs.body.split('\n').firstWhere(
+            (line) => line.contains('Authorize and Accept Invite</a>'));
```

Comment:
Could we make this less fragile by parsing the HTML? Or maybe just inserting an id... I imagine that this text might change over time.